script.lua

```diff
@@ -7729,122 +7732,82 @@ function onObjectDestroy(obj)
-                        posZ = pos[3],
-                        rotX = 0,
-                        rotY = 180,
-                        rotZ = 0,
-                        scaleX = 0.9,
-                        scaleY = 0.9,
-                        scaleZ = 0.9
-                    },
-                    Nickname = player_color.."'s "..card.getName().." Reminder",
-                    Tags = {"Destroy", "Mask", "Reminder Token"},
-                    Grid = false,
-                    Snap = false,
-                    CustomMesh = {
-                        MeshURL = "http://cloud-3.steamusercontent.com/ugc/1749061746121830431/DE000E849E99F439C3775E5C92E327CE09E4DB65/",
-                        DiffuseURL = "http://cloud-3.steamusercontent.com/ugc/2050879298352687582/0903B5F8D08AB12D8F4C05A703A9E193F049A702/",
-                        MaterialIndex = 1
-                    },
-                    LuaScript = "function onLoad()Wait.frames(function() self.UI.setXml(self.UI.getXml()) end, 2)end",
-                    XmlUI = "<Panel rotation=\"0 0 180\" width=\"350\" height=\"490\" position=\"31 86 -11\"><Mask image=\"CardMask\"><Image image=\""..imageURL.."\"/></Mask></Panel>",
-                }
-                if Tints[player_color] then
-                    local color
-                    if Tints[player_color].Token then
-                        color = Color.fromHex(Tints[player_color].Token)
-                    else
-                        color = Color.fromHex(Tints[player_color].Presence)
-                    end
-                    data.ColorDiffuse = {
-                        r = color.r,
-                        g = color.g,
-                        b = color.b
-                    }
-                end
-                spawnObjectData({data = data})
-            end,
-            false)
-    end
+    return nil
 end
-function spawnSpiritMarker(player_color, spirit)
-    local color = getSpiritColor({name = spirit.getName()})
-    if not color then
-        color = player_color
+function spawnMaskedReminder(color, obj, isMarker)
+    local objData = obj.getData()
+    local position = obj.getPosition()
+    local imageURL, maskImage
+    local panelWidth, panelHeight, panelX, panelY
+    local name, tags, scale
+
+    if obj.hasTag("Spirit") and objData.Name == "Custom_Tile" then
+        local spiritColor = getSpiritColor({name = obj.getName()})
+        if spiritColor then
+            color = spiritColor
+        end
+
+        position = position + Vector(0, 0.5, 0)
+        imageURL = objData.CustomImage.ImageSecondaryURL
+        if obj.hasTag("Lower Spirit Image") then
+            panelWidth, panelHeight, panelX, panelY = 378, 252, -97, -33
+            maskImage = "SpiritMask2"
+        else
+            panelWidth, panelHeight, panelX, panelY = 480, 320, -127, 46
+            maskImage = "SpiritMask"
+        end
+    elseif obj.type == "Card" then
+        position = position + Vector(0, 0.2, 0)
```

Comment:
not sure if this is intentional, but the value before was `0.02`

Reply:
Whoops, good catch! Fixed.